rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -1088,4 +1193,12 @@ private static ABICallElection getSampleElection(String functionName, AddressBas
         return new ABICallElection(authorizer, sampleVotes);
     }
 
+    private static Script createFederationScript() {
+        List<BtcECKey> fedSigners = BitcoinTestUtils.getBtcEcKeysFromSeeds(
+            new String[]{"fa06", "fa07", "fa08"}, true
+        );
+        Federation federation = createP2shErpFederation(federationConstants, fedSigners);
+        return federation.getP2SHScript();
+    }
+
```

Comment:
you could use the `P2shErpFederationBuilder` directly instead of creating this method

Reply:
Updated.